mirrord-agent/src/file.rs

```diff
@@ -21,7 +21,7 @@ use crate::{error::AgentError, runtime::get_container_pid, sniffer::DEFAULT_RUNT
 // `HashMap<_, RemoteFile`>, where `RemoteFile` is a struct that holds both the `File` + `PathBuf`.
 #[derive(Debug, Default)]
 pub struct FileManager {
-    pub open_files: HashMap<i32, File>,
+    pub open_files: HashMap<i32, (File, PathBuf)>,
```

Comment:
Why the change?

Reply:
I would suggest changing this tuple into a `RemoteFile` struct:

```rust
struct RemoteFile {
    inner: std::fs::File,
    path: PathBuf,
}
```

And then you could implement many of the file functions in a similar fashion to Rust's `std` (or just keep the current implementation of `FileManager` and we do something better later).